### PR TITLE
projectversion src repo: add latest release

### PR DESCRIPTION
**Pull request by szakalboss**
Signed-off-by: Karol Swiderski <szakalboss@gmail.com>

### CODE REVIEW

**Comment by szakalboss** [comment]
Example

image

**Comment by szakalboss** [comment]
It was added because our customers do not always understand how debian versioning works. Now they can see which versions will be used in their deployments

File: src/app/components/projectversion/projectversion-repo-list.html [hunk 1]
```diff
@@ -37,6 +37,15 @@ <h1><mat-icon>class</mat-icon>Project <a class="title" [routerLink]="['/project'
         </td>
     </ng-container>
 
+    <ng-container matColumnDef="latest_release">
+        <th mat-header-cell *matHeaderCellDef>Latest Release</th>
+        <td mat-cell *matCellDef="let element" [routerLink]="['/project', projectversion.project_name, projectversion.name, 'repo', element.id]">
+            <div *ngIf="element.latest_release" style="display: flex; align-items: center; font-weight: bold;">
+                {{element.latest_release.version}}
```
Comment (lbolla) [inline src/app/components/projectversion/projectversion-repo-list.html:44]: What if `element.laest_release` doesn't exist? It seems a possibility, given this code: https://github.com/molior-dbs/molior/pull/22/files#diff-a55367c6f8325b333ce41b5b5737f37ceb47a618502ce102443553200d35f0b0R268

If so, then won't `element.latest_release.version` raise an error?

Comment (szakalboss) [reply src/app/components/projectversion/projectversion-repo-list.html:44]: will add return None in molior API for such case

Comment (lbolla) [reply src/app/components/projectversion/projectversion-repo-list.html:44]: I am not sure I understand exactly what you are proposing, but even if `element.latest_release` is None, still trying to access `element.latest_release.version` would raise an error.

I think it's fine for `element.latest_release` to either be missing or be None: but in either case the frontend code above should check `if element.latest_release then... else...`.

Comment (szakalboss) [reply src/app/components/projectversion/projectversion-repo-list.html:44]: ngIf="element.latest_release" when it is None no access to .version -> we have div around

**Review by dario-gallucci — APPROVED** [review]

**Review by lbolla — APPROVED** [review]

**Review by randombenj — APPROVED** [review]

**Comment by szakalboss** [comment]
note: It will display "Current release" not "Latest release"

**Pull request closed without merging**